### PR TITLE
Documentation: stats/metrics reset on etcd restart

### PR DESCRIPTION
**Pull request by gyuho**
Addressing https://github.com/coreos/etcd/issues/4035.

/cc @xiang90 @joshix 

Thanks,

### CODE REVIEW

File: Documentation/api.md [hunk 1]
```diff
@@ -1052,6 +1052,7 @@ curl http://127.0.0.1:2379/v2/stats/self
 ### Store Statistics
 
 The store statistics include information about the operations that this node has handled.
+Note that v2 `store` is in-memory store. When a node stops, store statistics data will reset on restart.
```
Comment (xiang90) [inline Documentation/api.md]: also we need to update https://github.com/coreos/etcd/blob/master/Documentation/metrics.md

Comment (gyuho) [inline Documentation/api.md]: Ok

**Comment by gyuho** [comment]
@xiang90 Addressed. PTAL. Thanks,

File: Documentation/api.md [hunk 2]
```diff
@@ -1052,6 +1052,7 @@ curl http://127.0.0.1:2379/v2/stats/self
 ### Store Statistics
 
 The store statistics include information about the operations that this node has handled.
+Note that v2 `store` is in-memory store. When a node stops, store statistics will reset on restart.
```
Comment (xiang90) [inline Documentation/api.md]: Note that v2 `store Statistics` is stored in-memory.

**Comment by xiang90** [comment]
Technically correct. Defer to @joshix for doc/style review.

**Comment by joshix** [comment]
LGTM

File: Documentation/api.md [hunk 3]
```diff
@@ -1052,6 +1052,7 @@ curl http://127.0.0.1:2379/v2/stats/self
 ### Store Statistics
 
 The store statistics include information about the operations that this node has handled.
+Note that v2 `store Statistics` is stored in-memory. When a member stops, store statistics will reset on restart.
```
Comment (gyuho) [inline Documentation/api.md]: @xiang90 Just changed `node` to `member`. Will merge. Thanks,

**Pull request merged**